### ISSUE TITLE
Windows 10 allows windows to open outside if any screen and MDD editor stores coordinates and size of window for files.

### ISSUE THREAD

**Issue by tombensve** [tombensve/MarkdownDoc#31]
I have just myself run into a problem on Windows 10 when using the MarkdowndDoc editor.

I have 2 screens, my laptop and an external monitor. The external monitor was first places above the laptop screen. Running MarkdownDoc Editor with the editor window on the external screen (on top) saved the size and coordinates when saving the file. 

I then moved my external monitor and placed it on the right side instead and changed the monitor placement in windows 10. This gave it different coordinates. Opening the editor again opened the window at the previosuly saved coordinates which then apparently not are relative to the upper left corner of the monitor, but a point in a virtual coordinate system. So in other words the window opened outside of any screen!! This is clearly possible to do in Windows!! 

So the only solution is to stop saving position and only size. Not sure why I save positions, it feels kind of stupid right now. Though not as stupid as allowing windows to open outside of any screen.

**Comment by tombensve** [comment]
A temp quick fix if you like me have rearranged screens is to delete C:\Users\<you>\.mddoc folder.

**Comment by tombensve** [comment]
I have discovered that this problem does not only occur for MarkdownDoc on Windows 10. Many windows applications remember their coordinates in a badly thought out coordinate system used by Windows 10. When you move an external screen having a window on it to the left or right or below, windows will open the app at the same coordinates next time even if those coordinates does not map to a currently physical screen. I gave up and changed Windows settings to that when my laptop is connected to an external screen only the external screen is used, that is replacing the laptop screen as the main screen. When it is not connected to the external screen it uses the build in laptop screen. This seems to work without windows opening outside of any screen.

Windows 10 has become much better, but still has a way to go.
